src/app/loadout-builder/process/useProcess.ts

```diff
@@ -251,18 +264,22 @@ function groupItems(
       defs && upgradeSpendTierToMaxEnergy(defs, upgradeSpendTier, item) === 10
     }`;
 
-    if (raidCombatAndLegacyMods.length) {
+    if (activityMods.length) {
```

Comment:
This is also a slight logic change, I have removed combat mods here too as an optimisation. All armour has a combat mod socket now so we don't need to care about the tag (same handling as general mods really).